app/updater.js

```diff
@@ -47,13 +48,13 @@ exports.updateUrl = function (updates, platform, arch) {
     platform = platform + arch
   }
   platformBaseUrl = `${updates.baseUrl}/${Channel.channel()}/${version}/${platforms[platform]}`
-  console.log(platformBaseUrl)
   debug(`platformBaseUrl = ${platformBaseUrl}`)
   if (platform === 'darwin') {
     return platformBaseUrl
   } else {
-    debug(updates.winBaseUrl.replace('CHANNEL', Channel.channel()))
-    return updates.winBaseUrl.replace('CHANNEL', Channel.channel())
+    var windowsUpdateUrlWithArch = updates.winBaseUrl.replace('CHANNEL', Channel.channel()) + platforms[platform]
```

Comment:
Variable name a bit off since linux will go in this clause too won't it? Or if not why not?